### ISSUE TITLE
Card background colour not updating

### ISSUE THREAD

**Issue by tomlut** [home-assistant/home-assistant-polymer#3493]
**Home Assistant release with the issue:**

0.97.2
**Last working Home Assistant release (if known):**
0.96.5

**UI (States or Lovelace UI?):**

Lovelace

**Browser and Operating System:**

Windows 10, Chrome, Firefox.
**Description of problem:**

When the theme is changed with the `frontend.set_theme` service Lovelace card background colours do not update until a browser refresh or change of view.

**Javascript errors shown in the web inspector (if applicable):**
```

```

**Additional information:**

Before refresh:
![before refresh](https://user-images.githubusercontent.com/10679300/63138582-65253580-c01e-11e9-873b-14b433ba8d01.png)

After refresh:
![after refresh](https://user-images.githubusercontent.com/10679300/63138590-6e160700-c01e-11e9-8db3-d74e852c36a3.png)

**Comment by balloob** [comment]
Your screenshot only shows custom cards, those are not supported by us.

**Comment by tomlut** [comment]
Apologies, that is a very good point. Unfortunately the behaviour is the same for built in cards:

```
type: entities
entities:
  - entity: alarm_control_panel.house
  - entity: automation.7pm_abc_news_on_tv
title: Test
```

Before refresh:
![Annotation 2019-08-16 153420](https://user-images.githubusercontent.com/10679300/63145666-9d873c80-c03b-11e9-9583-afaed49248ab.png)


After:
![Annotation 2019-08-16 153445](https://user-images.githubusercontent.com/10679300/63145670-a1b35a00-c03b-11e9-9567-64baf25e6872.png)

**Comment by tomlut** [comment]
Additionally, if a theme is selected on the profile page it does not change the card background colour and no amount of browser cache clearing or forced refreshes help.

**Comment by tomlut** [comment]
Fixed with v0.98
